docs/configuration/arguments.md

```diff
@@ -393,9 +393,7 @@ Valid per-tenant limits are (with their corresponding flags for default values):
 
 - `max_series_per_query` / `-ingester.max-series-per-query`
```

Comment:
this is deprecated. The new one is -querier.max-fetched-series-per-query and is enforced at queriers,ruler and store-gateways